packages/bcer-retailer-app/app/src/views/GetHelp.tsx

```diff
@@ -42,6 +42,7 @@ export default function GetHelp() {
           and Sales Reports
         </div>
       </div>
+      <Typography variant="h6">If you experience any problems with the system, please send a request to <a href="https://mail.google.com/mail/?view=cm&fs=1&to=Vaping.info@gov.bc.ca" target="_blank" >Vaping.info@gov.bc.ca</a> </Typography>
```

Comment:
I think we should use a generic `mailto:` here instead of a google link in case the person does not use gmail.

`<a href="mailto:Vaping.info@gov.bc.ca">`